src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/AbstractFactory.php

```diff
@@ -65,7 +67,7 @@ public function create(ContainerBuilder $container, string $id, array $config, s
         }
 
         // create entry point if applicable (optional)
-        $entryPointId = $this->createEntryPoint($container, $id, $config, $defaultEntryPointId);
+        $entryPointId = $this->createDefaultEntryPoint($container, $id, $config, $defaultEntryPointId);
```

Comment:
This is the BC break that makes my code fail now. `AbstractFactory` is what we use in Connect Bundle: https://github.com/symfonycorp/connect-bundle/blob/master/src/DependencyInjection/Security/Factory/ConnectFactory.php

/cc @wouterj

So, even if I can probably fix it in Connect, I fear that we won't be alone by this problem. Anyone not using Guard is probably extending `AbstractFactory`. That means that we cannot mark `AbstractFactory` as `@internal`.

Reply:
Thanks for debugging!
That also means we have to find another method name for the `EntryPointInterface` interface, to not conflict with the abstract method in this class. I'll do a PR later this evening.